src/Security/Member.php

```diff
@@ -1044,7 +924,7 @@ public function onAfterWrite()
 
         Permission::reset();
 
-        if ($this->isChanged('Password')) {
+        if ($this->isChanged('Password') && static::config()->get('password_logging_enabled')) {
```

Comment:
Logging previous passwords allows systems to prevent password re-use - I'm not sure it's a security risk if they are salted and hashed... no more than storing passwords at all, anyway.